lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_update_release_notes.rb

```diff
@@ -41,7 +39,7 @@ def self.available_options
                                        env_name: 'FL_ANDROID_UPDATE_RELEASE_NOTES_FILE_PATH',
                                        description: 'The path to the release notes file to be updated',
                                        type: String,
-                                       default_value: File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'RELEASE-NOTES.txt')),
+                                       default_value: File.join('.', 'RELEASE-NOTES.txt')),
```

Comment:
At this point the `./` path prefix becomes unnecessary.
```suggestion
                                       default_value: 'RELEASE-NOTES.txt'),
```